src/mono/mono/metadata/native-library.c

```diff
@@ -1336,6 +1354,12 @@ lookup_pinvoke_call_impl (MonoMethod *method, MonoLookupPInvokeStatus *status_ou
 	}
 #endif
 
+#if defined(ENABLE_NETCORE) && !defined(NO_GLOBALIZATION_SHIM)
+	gpointer default_override = default_resolve_dllimport(new_scope, new_import);
```

Comment:
This means we will try the new resolve handler before `__Internal`, which is fine but worth noting for embedders.